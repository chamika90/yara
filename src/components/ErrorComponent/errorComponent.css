.container {
    width: 80%;
    height: 80%;
    padding: 20px;
}