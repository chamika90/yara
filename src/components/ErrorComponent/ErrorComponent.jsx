import React from "react";
import "./errorComponent.css";

export default function ErrorComponent({message}) {
  return (
    <div  className="container">
      <p data-test-id="errorMessage">{message}</p>
    </div>
  );
}