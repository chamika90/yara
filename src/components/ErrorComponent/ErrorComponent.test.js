import { render } from "@testing-library/react";
import ErrorComponent from "./ErrorComponent";

describe('render error component', () => {
    it('Should match snapshot', () => {
        const {toJSON}  = render(<ErrorComponent message={"Some thing went wrong."}/>);
        expect(toJSON).toMatchSnapshot();
    });
  });