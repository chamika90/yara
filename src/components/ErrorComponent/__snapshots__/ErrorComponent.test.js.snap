// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render error component Should match snapshot 1`] = `undefined`;
